### PR TITLE
Add RPC Server method ValidateAddress

### PR DESCRIPTION
**Pull request by im-kulikov**
- implement rpc method validateaddress (https://docs.neo.org/en-us/node/cli/2.9.4/api/validateaddress.html)
- add tests
- add to README.md

It's just implementation taken from [C#](https://github.com/neo-project/neo/blob/bbbc0cf5e401e425e7d749f26b23b824953da10a/neo/Network/RPC/RpcServer.cs#L566-L580)

### CODE REVIEW

**Comment by im-kulikov** [comment]
# IMPORTANT

merge https://github.com/CityOfZion/neo-go/pull/131 before

File: pkg/rpc/server.go [hunk 1]
```diff
@@ -181,9 +181,17 @@ Methods:
...
 		results = peers
 
-	case "validateaddress", "getblocksysfee", "getcontractstate", "getrawmempool", "getrawtransaction", "getstorage", "submitblock", "gettxout", "invoke", "invokefunction", "invokescript", "sendrawtransaction":
+	case "getblocksysfee", "getcontractstate", "getrawmempool", "getrawtransaction", "getstorage", "submitblock", "gettxout", "invoke", "invokefunction", "invokescript", "sendrawtransaction":
 		results = "TODO"
 
+	case "validateaddress":
+		param, exists := reqParams.ValueAt(0)
+		if !exists {
+			resultsErr = newError(-2146233086, http.StatusOK, "Index was out of range. Must be non-negative and less", "", nil)
```
Comment (AlexVanin) [inline pkg/rpc/server.go]: As @aprasolova suggested at https://github.com/CityOfZion/neo-go/pull/131 maybe we need constant for magic error code?

Comment (im-kulikov) [reply pkg/rpc/server.go]: @AlexVanin needs merge #131 before... than I remove this

Comment (AlexVanin) [reply pkg/rpc/server.go]: @im-kulikov no problem, I hope it will be merged soon

Comment (im-kulikov) [reply pkg/rpc/server.go]: I already fix that in #131

**Comment by hal0x2328** [comment]
Award im-kulikov 150 points

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/144dc06fbc54dd84165af08ddd9c01e8a720ca1bb05471dfd1af1503160f245f

**Pull request merged**